datacube/config.py

```diff
@@ -17,11 +18,13 @@
 #: - `datacube.conf`
 DEFAULT_CONF_PATHS = (
     '/etc/datacube.conf',
-    os.environ.get('DATACUBE_CONFIG_PATH'),
+    os.environ.get(ENVIRONMENT_VARNAME),
     os.path.expanduser("~/.datacube.conf"),
     'datacube.conf'
 )
 
+DEFAULT_ENV = 'default'
```

Comment:
Is the change from `[datacube]` to `[default]` for default configuration intentional?

Reply:
It is intentional, it may or may not be a good idea...

As far as I know, no one has been using the _environment_ stuff that Jeremy implemented yet, so I was wanting to clean it up a bit first.

Reply:
looks like it is, with `datacube` still used a fallback

Reply:
Using `[datacube]` was for backwards compatibility with previous releases: everyone's existing config files contain a `[datacube]` block already, so they'll work in the new release without modification (unless other fields have changed?).